plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1587,6 +1594,10 @@ void producer_plugin_impl::plugin_startup() {
          _irreversible_block_time = fc::time_point::maximum();
       }
 
+      if (!_is_savanna_active && irreversible_mode() && chain_plug->accept_transactions()) {
```

Comment:
Is this only for tests? All nodes in the field using this version of software (Rel 1.1.x) should have enabled Savanna. This is probably not needed.